runbotics/runbotics-desktop/src/action/sap/sap.action-handler.ts

```diff
@@ -91,35 +132,81 @@ export default class SapActionHandler extends StatefulActionHandler {
         return array;
     }
 
+    /**
+     *  @alias focus 
+     *  @name SAP: Focus
+     *  @description Selects an element and "highlights" it.
+     *  @param target
+     *  @example focus('wnd[0]/example/example')
+     */
     async focus(input: SapTypes.SAPFocusActionInput): Promise<SapTypes.SAPFocusActionOutput> {
         this.isApplicationOpen();
         this.session.FindById(input.target).setFocus();
         return {};
     }
 
+    /**
+     *  @alias disconnect 
+     *  @name SAP: Disconnect
+     *  @description Disconnects from instance on which the bot is currently logged in.
+     */
     async disconnect() {
         await this.session?.Parent.CloseConnection();
         this.session = null;
     }
 
+    /**
+     *  @alias sendVKey 
+     *  @name SAP: Send Virtual Key
+     *  @description Indicates keyboard shortcut to be clicked by bot.
+     *  @param virtualKey
+     *  @example sendVKey(Enter)
+     *  @see SendVKeyMapper - check out more examples
+     */
     async sendVKey(input: SapTypes.SAPSendVKeyActionInput): Promise<SapTypes.SAPSendVKeyActionOutput> {
         this.isApplicationOpen();
         this.session.FindById('wnd[0]').SendVKey(SendVKeyMapper[input.virtualKey]);
         return {};
     }
 
+    /**
+     *  @alias readText
+     *  @name SAP: Read text
+     *  @description Reads text from indicated element.
+     *  @param target
+     *  @example readText(wnd[0]/example)
+     *  @returns text value from chosen element
+     */
     async readText(input: SapTypes.SAPReadTextActionInput): Promise<SapTypes.SAPReadTextActionOutput> {
         this.isApplicationOpen();
         const result = this.session.FindById(input.target).text;
         return result ? result.__value : null;
     }
 
+    /**
+     *  @alias select
+     *  @name SAP: Select
+     *  @description In some SAP transactions, 'Select' action is utilized to navigate to another tab. 
```

Comment:
official description cannot be structured like this: `In some SAP transactions`, `Additionally`. Straightforward info about action. You can refer to exception like `FindById().select` which is something different than this method